demonstrations/tutorial_guide_to_pennylane_knowing_qiskit.py

```diff
@@ -0,0 +1,256 @@
+r"""Your guide to PennyLane if you know Qiskit
+===================================================
+
+Greetings, fellow quantum programmers 👋! Isn’t it such a wonderful world we live in, with so many
+different quantum software development kits (SDKs) at our fingertips? I remember the days of (messy)
+personalized code bases that took hours and hours to develop just *to be able* to start researching
+👴. Nowadays, those tools are there for us to access for free, being developed and maintained by
+`savvy open-source software developers <https://docs.pennylane.ai/en/stable/development/guide/contributing.html>`__ around the clock. Again, what a wonderful world!
+
+When it comes to quantum programming SDKs, `PennyLane <https://pennylane.ai/install/>`__ and `Qiskit <https://github.com/Qiskit/qiskit>`__ (``v1.0`` and ``<=v0.46``) are two
+of the most widely-used by the community. PennyLane has a few staples that make it so:
+
+- **Hardware-agnostic**: PennyLane has no opinions on what hardware or simulator backends you want
+  to use for your research. You can program an emporium of real hardware and simulator backends all
+  from the easy-to-use PennyLane API. This includes IBM’s hardware with the `PennyLane-Qiskit
+  plugin <https://docs.pennylane.ai/projects/qiskit/en/stable/>`__.
+   
+- **Everything is differentiable**: A quantum circuit in PennyLane is designed to behave like a
+  differentiable function, unlocking quantum differentiable programming and allowing to integrate
+  seamlessly with your favourite machine learning frameworks.
+   
+- **Community-focused**: Let’s face it, you’re going to get stuck at some point when you’re
+  researching or learning. That’s why we have a mandate to make our `documentation <https://docs.pennylane.ai/en/stable/index.html>`__easy to navigate,
```

Comment:
```suggestion
  researching or learning. That’s why we have a mandate to make our `documentation <https://docs.pennylane.ai/en/stable/index.html>`__ easy to navigate,
```
Whoops, missing a space, I think.